extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/ChangeRecord.java

```diff
@@ -154,6 +157,48 @@ public interface ChangeRecord {
     @Nonnull
     RecordPart value();
 
+    /**
+     * Returns the new value of the record. For <em>sync</em>, <em>insert</em> and <em>update</em> operations the value
+     * describes the database record as it looks AFTER the event, so the latest image, for <em>delete</em> it returns null.
+     */
+    @Nullable
+    default RecordPart newValue() {
+        try {
+            switch (operation()) {
+                case SYNC:
+                case INSERT:
+                case UPDATE: return value();
+                default: return null;
+            }
+        } catch (ParsingException e) {
+            throw rethrow(e);
+        }
+    }
```

Comment:
Why do you have the default implementation? There is only 1 implementation of the interface, and it overrides this method.

Reply:
In case if user implemented this own ChangeRecord. He shouldn't really, but it wasn't constrained in any way. If we don't consider it a public api, then I'll remove it.